websites/Y/YTyping/presence.ts

```diff
@@ -0,0 +1,202 @@
+import { ActivityType, Assets, getTimestamps } from 'premid'
+
+const presence = new Presence({
+  clientId: '1338529406905352212',
+})
+
+presence.on('UpdateData', async () => {
+  const privacy = await presence.getSetting<boolean>('privacy')
+
+  if (privacy) {
+    return presence.clearActivity()
+  }
```

Comment:
This isn't a very useful privacy system, since it is just another disable button. Remove the setting or change it to show generic messages instead.